Criterion/Report.hs

```diff
@@ -120,8 +137,10 @@ formatReport reports templateName = do
     templates <- getTemplateDir
     template <- includeTemplate (includeFile [templates]) template0
 
+
+    let reportJSON = TL.toStrict . TL.concatMap escapeJSON . encodeToLazyText
```

Comment:
Minor stylistic nitpick: move `reportJSON` to the `where` clause, as its definition does not depend on anything defined in the `do` block.